tag.php

```diff
@@ -0,0 +1,53 @@
+<?php
+	get_header();
+
+	$posts = get_posts( array(
+		'numberposts' => 10
+	) );
+
+	if( isset( $posts ) ) {
+		$first_post = array_shift( $posts );
+	}
+
+	$atts = array(
+		'posts_per_row' => 1,
+		'layout'        => 'vertical',
+		'layout__type'  => 'primary'
+	);
+?>
+
+<div class="container mt-4 mb-5 pb-sm-4">
+	<div class="row">
+		<div class="col-md-8">
+			<?php
+				if( isset( $posts ) && isset( $first_post ) ) {
+					echo today_post_list_display_feature( '', $first_post, $atts );
+
+					$atts['posts_per_row'] = 3;
+					echo today_post_list_display_feature( '', $posts, $atts );
```

Comment:
Using `today_display_feature_vertical()` directly here will be a little easier + give you more control over these list items, instead of accessing the post list plugin's layout functions directly.

e.g.

```php
<?php if ( $first_post ): ?>
	<?php echo today_display_feature_vertical( $first_post, array( 'layout__type' => 'primary' ) ); ?>
<?php endif; ?>

<?php if ( $posts ): ?>
<div class="row">
	<?php foreach ( $posts as $post ): ?>
		<div class="col-lg-4">
			<?php echo today_display_feature_vertical( $post ); ?>
		</div>
	<?php endforeach; ?>
</div>
<?php endif; ?>

<?php if ( ! $first_post && ! $posts ): ?>
<p>No results found.</p>
<?php endif; ?>
```